sdk/trace/batch_span_processor_test.go

```diff
@@ -560,24 +549,6 @@ func TestBatchSpanProcessorForceFlushCancellation(t *testing.T) {
 	}
 }
 
-func TestBatchSpanProcessorForceFlushTimeout(t *testing.T) {
-	tp := basicTracerProvider(t)
-	exp := newIndefiniteExporter(t)
-	bsp := NewBatchSpanProcessor(exp)
-	tp.RegisterSpanProcessor(bsp)
-	tr := tp.Tracer(t.Name())
-	_, span := tr.Start(context.Background(), "foo")
-	span.End()
-
-	// Add timeout to context to test deadline
-	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond)
-	defer cancel()
-
-	if got, want := bsp.ForceFlush(ctx), context.DeadlineExceeded; !errors.Is(got, want) {
-		t.Errorf("expected %q error, got %v", want, got)
-	}
-}
```

Comment:
This test was testing what would happen if SpanExporter.ExportSpans hangs. So no we are hanging because we are calling synchronously. I think this is fine as returning prematurely leaks a goroutine.